gce/jupyter-config.yaml

```diff
@@ -24,11 +24,11 @@ singleuser:
     enabled: true
 
 rbac:
-  enabled: false
+  enabled: true
 
 hub:
   extraConfig: |
-    c.KubeSpawner.singleuser_service_account = 'default'
+    c.KubeSpawner.singleuser_service_account = 'daskkubernetes'
```

Comment:
You could move this into the helm config itself instead of the `extraConfig` code as this config option is supported in `z2jh`.

```
singleuser:
    serviceAccountName: daskkubernetes
```

Reply:
Perhaps. This is working on our stack.

To be honest I'm not worried either way.

Reply:
Just a thought did you remove the ` c.KubeSpawner.singleuser_service_account = 'default'` line when setting within the `singleuser` section? Things in the extra config will override the helm config values.

Reply:
Pretty sure I did, but I can try again.